code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -240,7 +240,7 @@ var/const/MAX_ACTIVE_TIME = 400
 	if(M.stat == DEAD)
 		return 0
 
-	if(iscorgi(M) || ismonkey(M))
+	if(iscorgi(M) || issmall(M))

```

Comment:
Redundant and probably preventing monkies from protecting themselves by helmet. If that works at all.